tests/test_handlers.py

```diff
@@ -691,14 +691,14 @@ def mol_with_precomputed_charges():
     # fmt: off
     precomputed_charges = dict()
     precomputed_charges['ubuntu-18.04.4'] = np.array([
-        -3.385959, 0.26308782, 2.30802986, 1.11718023, -2.11460365,
-        0.9725527, -9.18744615, 0.04042971, -2.49344069, 1.87709392,
-        -1.64877796, 0.57591713, -1.68873618, 1.19414994, -1.04834368,
-        -1.52407053, 0.30670004, -1.26003975, -1.3233364, 33.79971321,
-        -10.86357005, -10.86357005, -1.72362592, -0.10184044, -1.47951542,
-        -1.72362592, -11.32255943, 2.74250182, 1.64583123, 1.55000221,
-        1.69816584, 1.83477824, 2.11436794, 1.99956175, 1.98447416,
-        2.86426252, 2.86426252])
+        -3.38603383, 0.26304593, 2.30806183, 1.11718269, -2.11466618,
+        0.97255779, -9.18759228, 0.04222331, -2.49355872, 1.87712273,
+        -1.64885893, 0.57591133, -1.68888899, 1.19419578, -1.04842043,
+        -1.52418996, 0.3067501, -1.2601894, -1.32341895, 33.79966262,
+        -10.86363258, -10.86363258, -1.72379533, -0.10198921, -1.47957496,
+        -1.72379533, -11.32267114, 2.74244351, 1.64587022, 1.54997692,
+        1.69815056, 1.83481513, 2.11438216, 1.99952399, 1.98443746,
+        2.86429782, 2.86429782])
     precomputed_charges['ubuntu-20.04.2'] = np.array([
```

Comment:
Haven't updated the ref values for `ubuntu-20.04.2` yet. If someone has these easily accessible, let me know, otherwise I'm tempted to remove this test altogether.

Reply:
These are the values I get:
[-3.3878352035023025, 0.26727080686425936, 2.3026757907523128, 1.1166965996609888, -2.107513358517747, 0.9612083769496058, -9.154301226767831, 0.03288091151318266, -2.4851042282886255, 1.8723247315824088, -1.6547659251138624, 0.5817522843277193, -1.6897608745990291, 1.191577230801982, -1.0478643525397062, -1.5200599330326403, 0.3044723600341638, -1.2594928059462964, -1.326439016721788, 33.755887188134906, -10.851852320219985, -10.851852320219985, -1.724017252391697, -0.10285399008405359, -1.4786711130785013, -1.724017252391697, -11.319224351341875, 2.7444770823135793, 1.6425628940842003, 1.5516172308731973, 1.6987242017361495, 1.8342000296623708, 2.114672322516452, 1.9941244248440126, 1.9856416523206082, 2.866430198256134, 2.866430198256134]

Reply:
Thanks!